DevCycle.SDK.Server.Cloud.Example/DevCycle.SDK.Server.Cloud.Example.csproj

```diff
@@ -13,7 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="DevCycle.SDK.Server.Cloud" Version="1.4.0" />
+    <PackageReference Include="DevCycle.SDK.Server.Cloud" Version="1.10.0" />
```

Comment:
anyway to just set this to use the latest so we don't have to remember to update it?

Reply:
good question.  i'll look into that

Reply:
no it looks like you need to be explicit with the version in this csproj files, there is no concept of "latest"